.github/workflows/merge-flow.yaml

```diff
@@ -94,6 +94,11 @@ jobs:
       - uses: actions/setup-go@v2
         with:
           go-version: ${{ inputs.go-version }}
+      - name: Remove dependabot configuration
+        continue-on-error: true
+        run: |
+          rm -r .github/dependabot.*
```

Comment:
did you mean delete `-r` and keep `-f` ?

Reply:
I suppose use force, but since I have `continue-on-error: true`, I can remove `-f` too.